functions/backup.bash

```diff
@@ -377,7 +377,8 @@ mirror_SD() {
   fi
   if [[ "$1" == "raw" ]]; then
     echo "Creating a raw partition copy, be prepared this may take long such as 20-30 minutes for a 16 GB SD card"
-    if ! cond_redirect dd if="${src}" bs=1M of="${dest}"; then echo "FAILED (raw device copy)"; return 1; fi
+    if ! cond_redirect dd if="${src}" bs=1M of="${dest}"; then echo "FAILED (raw device copy)"; dirty="yes"; fi
+    if ! cond_redirect echo "y" | set-partuuid "${dest}" random; then echo "FAILED (set random PARTUUID)"; dirty="yes"; fi
```

Comment:
don't use `echo "y"` rather use the `yes` command

Reply:
didn't know it. at your own risk ...

Reply:
Yep